docs/Data_Portal/Users_Guide/cohort_builder.md

```diff
@@ -90,3 +90,40 @@ Users can then perform the following actions:
 
 * Download files associated with the cohort from the [Repository](Repository.md)
 * Analyze data from the cohort in the [Analysis Center](analysis_center.md)
+
+## Cohort Types
+
+Depending on how they are modified or created, cohorts can have different types of filters and thus behave differently:
+
+* __Custom Queries__
+* __Specific List of Cases__
+
+### Custom Queries
+
+Examples of cohorts with custom queries would be:
```

Comment:
Maybe add some initial context to what is meant by a custom query cohort; if I understand correctly, maybe something like, "One way cohorts can be built is by customizing filters and queries in the Cohort Builder Tool or in the Query Expressions section. Examples of cohorts... "

Reply:
will do 👍